articles/active-directory/develop/quickstart-v2-java-webapp.md

```diff
@@ -120,28 +111,23 @@ attributes.
 
 #### Step 4: Run the code sample
 
 To run the project, you can either:
 
-- Run it directly from your IDE by using the embedded spring boot server
-- or package it to a WAR file using [maven](https://maven.apache.org/plugins/maven-war-plugin/usage.html) and deploy it to a J2EE container solution such as [Apache Tomcat](http://tomcat.apache.org/).
+- Run it directly from your IDE by using the embedded spring boot server or
+- Package it to a WAR file using [maven](https://maven.apache.org/plugins/maven-war-plugin/usage.html) and deploy it to a J2EE container solution such as [Apache Tomcat](http://tomcat.apache.org/).
 
 ##### Running from IDE
 
 If you are running the web application from an IDE, click on run, then navigate to the home page of the project. For this sample, the standard home page URL is <http://localhost:8080>
 
-1. The front page contains a **sign-in** button. Click on the **sign-in** button to redirect to Azure Active Directory. The
-user will be prompted for their credentials.  
-1. After successfully authenticating on Azure Active Directory, they will be redirected to
-*http://localhost:8080/msal4jsamples/secure/aad*. They are officially signed in to the application, and the page should show
-information for the signed in account. It will also contain buttons for:
-    - *Sign Out*: Will sign out the current user from the application, and redirect them the home page.
-    - *Show User Info*: Will acquire a token for the Microsoft Graph, then call the Microsoft Graph with the token attached to
-    the request to get the basic information of the signed-in user.
+1. The front page contains a **sign-in** button. Click on the **sign-in** button to redirect to Azure Active Directory. The user will be prompted for their credentials.  
+1. After successfully authenticating on Azure Active Directory, they will be redirected to *http://localhost:8080/msal4jsamples/secure/aad*. They are officially signed in to the application, and the page should show information for the signed in account. It will also contain buttons for:
+    - *Sign Out*: Signs the current user out of the application and redirects them to the home page.
+    - *Show User Info*: Acquires a token for Microsoft Graph and calls Microsoft Graph with a request containing the token, which returns basic information about the signed-in user.
 
 >[!IMPORTANT]
 This quickstart application uses a client secret to identify itself as confidential client. Because the client secret is added as a plain-text to your project files. For security reasons, it is recommended that you use a certificate instead of a client secret before considering the application as production application. For more information on how to use a certificate, see these [instructions](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-certificate-credentials).
```

Comment:
Incomplete sentence. Because the client secret is added as plain-text, what?

Reply:
Edited it as "Because the client secret is added as a plain-text to your project files, for security reasons it is..... "

Reply:
nice.